mr/src/main/java/org/elasticsearch/hadoop/cfg/ConfigurationOptions.java

```diff
@@ -198,6 +198,9 @@
     String ES_SPARK_DATAFRAME_WRITE_NULL_VALUES = "es.spark.dataframe.write.null";
     String ES_SPARK_DATAFRAME_WRITE_NULL_VALUES_DEFAULT = "false";
 
+    String ES_SPARK_FIND_SLICE_PARTITIONS = "es.find.slice.partitions";
```

Comment:
Lets move this up to the `/** Input options */` section and rename the property to something like "es.input.use.sliced.scroll" or ""es.input.use.sliced.partitions".